### ISSUE TITLE
SHA1 named export not found

### ISSUE THREAD

**Issue by rohannair** [easyblockshq/easyblocks#33]
# Steps to reproduce
1. Set up new Remix app using Remix/Vite generator
2. `yarn add @easyblocks/editor @easyblocks/core`
3. Paste code from Getting Started guide into a new remix page
4. `yarn dev`

## Expected
- Remix app starts

## Actual
- App crashes with following error:

```shell
[vite] Internal server error: Named export 'SHA1' not found. The requested module 'crypto-js' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'crypto-js';
const { SHA1 } = pkg;

      at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
      at async ModuleJob.run (node:internal/modules/esm/module_job:190:5)
```

## Other relevant data
tsconfig.json:
```json
{
  "include": [
    "**/*.ts",
    "**/*.tsx",
    "**/.server/**/*.ts",
    "**/.server/**/*.tsx",
    "**/.client/**/*.ts",
    "**/.client/**/*.tsx"
  ],
  "compilerOptions": {
    "lib": ["DOM", "DOM.Iterable", "ES2022"],
    "types": ["@remix-run/node", "vite/client"],
    "isolatedModules": true,
    "esModuleInterop": true,
    "jsx": "react-jsx",
    "module": "ESNext",
    "moduleResolution": "Bundler",
    "resolveJsonModule": true,
    "target": "ES2022",
    "strict": true,
    "allowJs": true,
    "skipLibCheck": true,
    "forceConsistentCasingInFileNames": true,
    "baseUrl": ".",
    "paths": {
      "~/*": ["./app/*"]
    },

    // Vite takes care of building everything, not tsc.
    "noEmit": true
  }
}

```

package.json (truncated for sanity):
```json
{
  "version": "1.0.0",
  "private": true,
  "sideEffects": false,
  "type": "module",
  "scripts": {
    "build": "remix vite:build",
    "dev": "remix vite:dev",
    "start": "remix-serve ./build/server/index.js",
    "typecheck": "tsc"
  },
  "dependencies": {
    "@easyblocks/core": "^1.0.4-alpha.0",
    "@easyblocks/editor": "^1.0.4-alpha.0",
    "react": "^18.3.0-canary-2f8f77602-20240229",
    "react-dom": "^18.3.0-canary-2f8f77602-20240229",
  },
}

```

vite config:
```typescript
import { vitePlugin as remix } from '@remix-run/dev'
import { installGlobals } from '@remix-run/node'
import UnoCSS from 'unocss/vite'
import { defineConfig } from 'vite'
import tsconfigPaths from 'vite-tsconfig-paths'

installGlobals()

export default defineConfig({
  plugins: [remix(), UnoCSS(), tsconfigPaths()],

  server: {
    port: 8088,
    fs: {
      // Restrict files that could be served by Vite's dev server.  Accessing
      // files outside this directory list that aren't imported from an allowed
      // file will result in a 403.  Both directories and files can be provided.
      // If you're comfortable with Vite's dev server making any file within the
      // project root available, you can remove this option.  See more:
      // https://vitejs.dev/config/server-options.html#server-fs-allow
      allow: ['app'],
    },
    hmr: {
      overlay: false,
    },
  },
})
```

**Comment by rohannair** [comment]
@r00dY this is _not_ the issue I had mentioned to you on Monday, but is in a similar area around cjs vs mjs modules.

**Comment by rohannair** [comment]
That is the version I am on.

**Comment by pawel-commerce-studio** [comment]
Hi @rohannair!

Thanks for reporting this. We've shipped a fix in version `1.0.5-alpha.0`. Let us know if everything went well

**Comment by rohannair** [comment]
Yes, that fixed it!
